dashboard/http_server_head.py

```diff
@@ -128,6 +129,17 @@ def get_address(self):
         assert self.http_host and self.http_port
         return self.http_host, self.http_port
 
+    @aiohttp.web.middleware
+    async def path_clean_middleware(self, request, handler):
```

Comment:
Would it be possible to normalized all path values before passing it forward in the routing handler? (can we even do that in a middleware?)

Reply:
I was thinking that since these APIs lean on the request path itself and not a parameter or data field; if we could normalize the `request.Path` field itself before it was handled by the router/mux, then this would 404 instead of getting into the final handler.

That said, that almost certainly isn't something we can do from within Starlette middleware? Noop this idea xD